metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -276,6 +240,19 @@ private String getAssemblyAccession(ReferenceSequenceType referenceSequenceType)
         return null;
     }
 
+    private Set<String> getSequenceAndTsaAccessions(ReferenceSequenceType referenceSequenceType) {
+        Set<String> sequenceAndTsaAccessions = new HashSet<>();
```

Comment:
No, accessions in `SEQUENCEArray` cannot be duplicated because it would not have any biological meaning. _Even if_ for some EVA/EGA records an accession is present multiple times (due to a malformed XML for example), it should still be collapsed into a single instance, hence using `HashSet`